packages/frontend/pages/positions.tsx

```diff
@@ -225,355 +227,393 @@ export function Positions() {
           <Typography color="primary" variant="h6">
             Your Positions
           </Typography>
-          <div style={{ display: 'flex' }}>
-            <Typography component="span" color="textSecondary">
-              ETH Price:{' '}
-            </Typography>
+          {supportedNetwork && (
```

Comment:
Same here, What if we forget this in some places? Instead of manually adding this check in every possible component we should just reset the address, so there will be no data. Remembering this is will definitely be difficult.

@yichiehliu @EstherSeyi @adam5909099 Thoughts